### PR TITLE
feat: add tabs to subscribers page

### PR DESCRIPTION
**Pull request by lucianaDilizio**
### Tabs

- [X] Add tabs to subscribers page. 

Link: https://cdn.fromdoppler.com/doppler-webapp/demo-build2518/#/subscribers/lgarcia@makingsense.com/history

subscriberTabs

### CODE REVIEW

**Comment by lucianaDilizio** [comment]
Test this please :)

File: src/i18n/en.js [hunk 1]
```diff
@@ -321,6 +321,7 @@ deletion, security, cross-border data transfers and other issues.
     permission_description: 'Custom text',
     permission_name: 'Field Name',
     permission_value: 'Value',
+    tab_description: 'Subscriber GDPR state',
```
Comment (cbernat) [inline src/i18n/en.js:324]: Have you checked this with @mrunge86 ?

File: src/i18n/es.js [hunk 2]
```diff
@@ -339,6 +340,7 @@ eliminación, seguridad, transferencias transfronterizas y otros temas.
     empty_data: 'Hasta el momento no hay Campañas enviadas',
     subscriber_breadcrumb: 'Suscriptores',
     subscriber_breadcrumb_url: `${urlMasterSubscriber}`,
+    tab_description: 'Comportamiento histórico del suscriptor',
```
Comment (cbernat) [inline src/i18n/es.js:343]:
```suggestion
    tab_description: 'Comportamiento histórico del Suscriptor',
```

File: src/i18n/en.js (hunk 1 above)
Comment (lucianaDilizio) [reply src/i18n/en.js:324]: No I didn't review the content yet, I'm reusing what we already had.

File: src/components/Reports/Subscribers/Subscribers.js [hunk 3]
```diff
@@ -33,6 +34,18 @@ const Subscribers = ({ dependencies: { dopplerApiClient } }) => {
...
+      url: `/subscribers/${email}/history`,
+      descriptionKey: 'subscriber_history.tab_description',
+      active: section === 'history',
+    },
+    {
+      url: `/subscribers/${email}/gdpr`,
+      descriptionKey: 'subscriber_gdpr.tab_description',
+      active: section === 'gdpr',
+    },
+  ];
```
Comment (andresmoschini) [inline src/components/Reports/Subscribers/Subscribers.js:48]: In my opinion, you should reuse sections object

File: src/components/shared/Tabs/Tabs.js [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+import { Link } from 'react-router-dom';
+
+export const Tabs = ({ tabsProperties }) => {
+  return (
+    <nav className="tabs-wrapper">
+      <ul className="tabs-nav" data-tab-active="1">
+        {tabsProperties.map((tab) => (
+          <li className="tab--item">
+            <Link to={tab.url} className={tab.active ? 'tab--link active' : 'tab--link'}>
+              <FormattedMessage id={tab.descriptionKey} />
```
Comment (andresmoschini) [inline src/components/shared/Tabs/Tabs.js:12]: In my opinion, translation should be out of the scope of this component

File: src/i18n/en.js (hunk 1 above)
Comment (andresmoschini) [reply src/i18n/en.js:324]: What do you think about reusing _dapeg_description_?

File: src/components/Reports/Subscribers/Subscribers.js (hunk 3 above)
Comment (andresmoschini) [reply src/components/Reports/Subscribers/Subscribers.js:48]: if you need, you can transform (or map) _sections_ object to generate the required structure for _Tabs_ component.

Comment (lucianaDilizio) [reply src/components/Reports/Subscribers/Subscribers.js:48]: I don't realize how to do this. _sections_ is an object and _Tabs_ component expect an array.

Comment (andresmoschini) [reply src/components/Reports/Subscribers/Subscribers.js:48]: But as you can see, there is relation between them, right? DRY

Let my know if you need help in the conversion.

Comment (lucianaDilizio) [reply src/components/Reports/Subscribers/Subscribers.js:48]: I will try to find a way by myself.

**Comment by lucianaDilizio** [comment]
Test this please :)

**Comment by lucianaDilizio** [comment]
Test this please

**Comment by lucianaDilizio** [comment]
test this please 😠

**Comment by lucianaDilizio** [comment]
test this please

**Comment by lucianaDilizio** [comment]
test this please

**Comment by lucianaDilizio** [comment]
TEST THIS PLEASE

**Comment by lucianaDilizio** [comment]
test this please

**Pull request merged**